docs/USAGE.md

```diff
@@ -2,37 +2,95 @@
 
-When designing content to be displayed in an OVE environment, be aware of the [potential pitfalls](./PITFALLS.md).
+You will need to modify the [`Spaces.json`](https://github.com/ove/ove/blob/master/packages/ove-core/src/client/Spaces.json) file and introduce a new `space` in OVE, before it can be used in a production setting.
 
 ## Launching browsers
 
-On each monitor that makes up the display you will need to launch a browser window, and open the correct URL in each. This has the form: ``http://<ove-hostname>:<ove-port>/view.html?oveViewId=<space>-{id}``
-where ``{id}`` is the index of the ``client`` associated with the browser window in the space's definition in ``Spaces.json``.
+In order to use OVE, you will need to open the URLs of the `clients` in a web browser. Each URL has the form: `http://OVE_CORE_HOST:PORT/view.html?oveViewId={SPACE}-{ID}`. The values for `OVE_CORE_HOST` and `PORT` must be set as explained in the [OVE Installation Guide](./INSTALLATION.md#running-ove). The value for `SPACE` would be the name of the `space` used, such as `LocalNine`, `LocalFour`, or a name of a new `space` that has been defined in the [`Spaces.json`](https://github.com/ove/ove/blob/master/packages/ove-core/src/client/Spaces.json) file. The value for `ID` is the index of the `client` associated with the browser window, in the definition of the `space` in [`Spaces.json`](https://github.com/ove/ove/blob/master/packages/ove-core/src/client/Spaces.json) file.
+
+It is recommended to use OVE with Google Chrome, as this is the web browser used for development and in production at the [Data Science Institute](http://www.imperial.ac.uk/data-science/). However, it should also be compatible with other modern web browsers: if you encounter any browser-specific bugs please [report them as an Issue](https://github.com/ove/ove-apps/issues).
+
+To test OVE functionality, you can simply open four web browser windows (or tabs) with the following URLs to create the `LocalFour` space:
+
+```text
+http://OVE_CORE_HOST:PORT/view.html?oveViewId=LocalFour-0
+http://OVE_CORE_HOST:PORT/view.html?oveViewId=LocalFour-1
+http://OVE_CORE_HOST:PORT/view.html?oveViewId=LocalFour-2
+http://OVE_CORE_HOST:PORT/view.html?oveViewId=LocalFour-3
+```
+
+However, in a much larger OVE installation with many monitors, opening browser windows may be sufficiently time-consuming for automation to be worthwhile.
+
+In a Windows environment, a [PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/powershell-scripting) script can be used to launch full-screen browsers with the correct URLs. In a Linux or MacOS environment, a [Bash](https://linux.die.net/man/1/bash) script can be used to launch full-screen browsers with the correct URLs. In a MacOS environment, [AppleScript](https://developer.apple.com/library/archive/documentation/AppleScript/Conceptual/AppleScriptLangGuide/introduction/ASLR_intro.html) can be used as an alternative to [Bash](https://linux.die.net/man/1/bash).
```

Comment:
```suggestion
On Windows, a [PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/powershell-scripting) script can be used to launch full-screen browsers with the correct URLs. On Linux, a [Bash](https://linux.die.net/man/1/bash) script can be used. On MacOS, either [AppleScript](https://developer.apple.com/library/archive/documentation/AppleScript/Conceptual/AppleScriptLangGuide/introduction/ASLR_intro.html) or [Bash](https://linux.die.net/man/1/bash) could be used.
```

Reply:
Actually speaking AppleScript is not the first preference on a Mac. I will therefore reverse the order and include this.